src/panel/service_panel.js

```diff
@@ -1,47 +1,81 @@
-import ServicePanelView from '../views/service_panel.dot';
-import Panel from '../libs/panel';
-import nconf from '@qwant/nconf-getter';
+/* globals _ */
+import React from 'react';
+import ReactDOM from 'react-dom';
 import CategoryService from '../adapters/category_service';
 
-export default class ServicePanel {
-  constructor() {
-    this.panel = new Panel(this, ServicePanelView);
-    this.categories = CategoryService.getCategories();
-    this.mustDeployCategories = this.categories.length > 8;
-    this.isDeployed = false;
-    this.isDirectionActive = nconf.get().direction.enabled;
-    this.active = true;
-  }
+// Returns a function, which when called will navigate to the url
+const navTo = url => () => {
+  window.app.navigateTo(url);
+};
 
-  toggleCategories() {
-    this.isDeployed = !this.isDeployed;
-    this.panel.update();
-  }
+const CategoryButton = ({ category }) => {
+  return <button
+    className="service_panel__category"
```

Comment:
JSX is mostly XHTML, which some particularities. One is that some attribute names which are also JS reserved terms cannot be used. The most common one is `class` which is replaced by `className`.